lib/src/ruby/module.cc

```diff
@@ -135,14 +136,7 @@ namespace facter { namespace ruby {
         _collection(facts),
         _loaded_all(false)
     {
-#ifdef FACTER_RUBY
```

Comment:
This was the wrong place - and caused the failure in Jenkins CI - because Ruby is always first initialized in facter::ruby::initialize.